Common/Utilities.cs

```diff
@@ -1,53 +1,46 @@
 ﻿// Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License. See License.txt in the project root for license information.
 
+using Azure.Messaging.ServiceBus;

```

Comment:
I don't understand what the huge list here has to do with the stated goal of this sample.   The sample should be focused on managing Service Bus.   We shouldn't need to pull in all of these other services.

Reply:
Discussed in below. After removed redundant method, most of this list were gone.